sjust/recipes/00-default.just

```diff
@@ -259,6 +259,73 @@ sparkdock-menubar-install:
     echo "✅ Sparkdock menu bar app rebuilt and installed successfully!"
     echo "💡 The app should now be running via LaunchAgent"
 
+# Fix Xcode command line tools and rebuild menubar application.
+[group('sparkdock')]
+sparkdock-menubar-fix-and-install:
+    #!/usr/bin/env bash
+    set -euo pipefail
+    echo "🔧 Fixing Xcode command line tools and rebuilding Sparkdock menu bar application..."
+    echo ""
+    echo "This will fix common Swift compilation issues by reinstalling Xcode command line tools."
+    echo "You will be prompted for your password and may need to interact with the installer."
+    echo ""
+
+    # Check if we're in a CI environment
+    if [[ -n "${CI:-}" || -n "${GITHUB_ACTIONS:-}" || -n "${NON_INTERACTIVE:-}" ]]; then
+        echo "❌ This task cannot run in CI/non-interactive environments"
+        echo "💡 Run this task manually on your local machine if you encounter compilation issues"
+        exit 1
+    fi
+
+    # Confirm with user
+    read -p "🤔 Do you want to proceed with removing and reinstalling Xcode command line tools? (y/N): " -n 1 -r
+    echo
+    if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+        echo "❌ Operation cancelled"
+        exit 1
+    fi
+
+    echo "🗑️  Removing existing Xcode command line tools..."
+    if [[ -d "/Library/Developer/CommandLineTools" ]]; then
+        sudo rm -rf /Library/Developer/CommandLineTools
+        echo "✅ Existing command line tools removed"
+    else
+        echo "ℹ️  No existing command line tools found"
+    fi
```

Comment:
do not print the echo in the else statement, just inform the user of the things you do, not the things are not doing.